editor/scene_tree_dock.cpp

```diff
@@ -93,6 +128,8 @@ void SceneTreeDock::input(const Ref<InputEvent> &p_event) {
 			pending_click_select = nullptr;
 		}
 	}
+
+	_handle_hover_to_inspect();
```

Comment:
You can use `get_viewport()->gui_is_dragging()` to handle this only when dragging (as otherwise it's irrelevant).